cmd/argocd/commands/cluster.go

```diff
@@ -122,6 +127,7 @@ func NewClusterAddCommand(clientOpts *argocdclient.ClientOptions, pathOpts *clie
 	command.PersistentFlags().StringVar(&pathOpts.LoadingRules.ExplicitPath, pathOpts.ExplicitFileFlag, pathOpts.LoadingRules.ExplicitPath, "use a particular kubeconfig file")
 	command.Flags().BoolVar(&inCluster, "in-cluster", false, "Indicates Argo CD resides inside this cluster and should connect using the internal k8s hostname (kubernetes.default.svc)")
 	command.Flags().BoolVar(&upsert, "upsert", false, "Override an existing cluster with the same name even if the spec differs")
+	command.Flags().StringVar(&serviceAccount, "service-account", "", "System namespace service account to use for kubernetes resource management. If not set then default \"argocd-manager\" SA will be created")
```

Comment:
Can you please either use reference to `clusterauth.ArgoCDManagerServiceAccount` in the help text, instead of hard coded name? We don't expect the name of the SA to change, but should it happen, it'd be nice to have the help text automatically adapted. Or just use "If not set, a dedicated SA will be created and authorized" or something along these lines, without specifying the name.

Reply:
@jannfis fair enough, adjusted